### PR TITLE
feat: Send error to channel with default error handlers

### PR DESCRIPTION
**Pull request by LordOfPolls**
## What type of pull request is this?

- [x] Non-breaking code change
- [ ] Breaking code change
- [ ] Documentation change/addition
- [ ] Tests change

## Description

The default error handler now sends the error to the channel the command was ran in

## Changes

- Added send call to error handler
- Organised imports and converted relative imports 

## Checklist

- [ ] I've formatted my code with [Black](https://black.readthedocs.io/en/stable/)
- [ ] I've ensured my code works on `Python 3.10.x`
- [ ] I've tested my code

### CODE REVIEW

**Comment by LordOfPolls** [comment]
image

**Comment by AstreaTSS** [comment]
Couldn't this have potentially dangerous information, like bot tokens or API keys? I think it's a bit unwise to show off the full traceback.

**Comment by LordOfPolls** [comment]
Let them leak dammit \s 

Jokes aside, the trace back will leak the token/api key *if* the user was daft enough to hardcode their token/keys in their src files. *Not* showing tracebacks limits the amount of usable information that can be gleamed from the error message. Bot developers could implement their own error messages that allow for looking up errors, but the base lib cant have that functionality. 

I do honestly think that vulnerability is something the bot developer should mitigate, not us. I'd also be inclined to point out other libraries send tracebacks by default too

**Comment by LordOfPolls** [comment]
As the bare-minimum ill filter the bot token. I will make it clear to those reading this 

**DO NOT RELY ON THIS TO KEEP YOUR TOKENS SAFE**
Do the bare minimum security on your projects and put the token in a text file or something

**Comment by leestarb** [comment]
I suggest moving this to the debug scale
+
I dont think this is really useful

File: dis_snek/client/client.py [hunk 1]
```diff
@@ -478,7 +481,17 @@ async def on_command_error(self, ctx: Context, error: Exception, *args, **kwargs
         Override this to change error handling behavior
 
         """
-        return await self.on_error(f"cmd /`{ctx.invoked_name}`", error, *args, **kwargs)
+        await self.on_error(f"cmd /`{ctx.invoked_name}`", error, *args, **kwargs)
+        try:
+            out = "".join(traceback.format_exception(error))
```
Comment (leestarb) [inline dis_snek/client/client.py]: This needs the [:1990] slice ig

**Review by leestarb — CHANGES_REQUESTED** [review]

**Comment by LordOfPolls** [comment]
> I suggest moving this to the debug scale + I dont think this is really useful

I disagree on both counts

File: dis_snek/client/client.py (hunk 1 above)
Comment (LordOfPolls) [reply dis_snek/client/client.py]: Agreed, adding now

**Comment by LordOfPolls** [comment]
image

**Comment by LordOfPolls** [comment]
Requesting reviews

**Comment by leestarb** [comment]
I suggest moving this logic to the utils package

**Comment by LordOfPolls** [comment]
Moving which logic? The logic for the 3 different embeds? 

Is that really utility worthy? It's 3 if statements and some embeds. Surely if youre implimenting your own error handler you'd want to create your own embeds, which would defeat the point of making the logic a util

**Review by silasary — APPROVED** [review]

**Pull request merged**